### ISSUE TITLE
Input normalization

### ISSUE THREAD

**Issue by oyxhust** [yqyao/SSD_Pytorch#3]
Hello, thanks for your great work! I have a question about the input normalization.

Usually, the input will do the "ToTenser" function which will make the input range to [0, 1]. In your implement, I find that the input will reduce the mean then it will range to about [-127, 127]. Is that a trick to train SSD without to normalize to [0, 1].

**Comment by yqyao** [comment]
No, I just do it same with original caffe implementation @oyxhust . In caffe, we don't need to normalized input to [0, 1].
